plugins/org.jkiss.dbeaver.ext.postgresql.ui/src/org/jkiss/dbeaver/ext/postgresql/ui/PostgreConnectionPage.java

```diff
@@ -250,6 +256,11 @@ public void loadSettings() {
             }
             dbText.setText(databaseName);
         }
+        if (showNonDefault != null) {
+            showNonDefault.setSelection(
+                CommonUtils.getBoolean(connectionInfo.getProviderProperty(PostgreConstants.PROP_SHOW_NON_DEFAULT_DB),
+                    DBWorkbench.getPlatform().getPreferenceStore().getBoolean(PostgreConstants.PROP_SHOW_NON_DEFAULT_DB)));
```

Comment:
```suggestion
            showNonDefault.setSelection(CommonUtils.getBoolean(
                connectionInfo.getProviderProperty(PostgreConstants.PROP_SHOW_NON_DEFAULT_DB),
                DBWorkbench.getPlatform().getPreferenceStore().getBoolean(PostgreConstants.PROP_SHOW_NON_DEFAULT_DB)
            ));
```
Not sure if it's better, but this way it's clearer that these properties are passed in the `CommonUtils.getBoolean` method